### DISCUSSION TITLE
How to use different strategies between training and validation loops?

### DISCUSSION THREAD

**Discussion by Meehai** [Lightning-AI/pytorch-lightning#20287] [DDP / multi-GPU / multi-node]
Hello, I'd like to do the following workflow and was wondering if it's possible with the current `pl.Trainer`:
- train set using DDP (on >=2 GPUs) -> dataset is split and metrics are shared and synced using `self.log(..., sync_dist=True)` in the `on_train_batch/epoch_end` methods
- validation set using no strategy, computed on the first GPU only -> full validation set remains in GPU 0

However, as far as I know, the trainer object receives a single `strategy` parameter (i.e. there's no `validation_strategy` parameter). Is there any way to achieve what I want?
